akka-docs/src/main/paradox/.htaccess

```diff
@@ -27,3 +27,5 @@ RedirectMatch 301 ^(.*)/stream/operators/Source-or-Flow/apply\.html$ $1/stream/o
 RedirectMatch 301 ^(.*[^d])/guide/(.*) $1/typed/guide/$2
 
 RedirectMatch 301 ^(.*)/additional/rolling-deploys\.html$ $1/additional/rolling-updates.html
+
+RedirectMatch 301 ^(.*)/howto\.html$ https://doc.akka.io/docs/akka/2.5/howto.html
```

Comment:
Should we really auto-redirect to another version of the docs?

Reply:
my thinking was if someone was explicitly linking to the page that's the best we can do... of course we should make sure we're not linking to that page ourselves anymore - at least within the Akka docs paradox makes sure of that.

Reply:
sounds ok to me